theano/gpuarray/blas.py

```diff
@@ -38,6 +39,7 @@ class GpuGemv(BlasOp):
     Gemv on the GPU.
 
     """
+    params_type = ParamsType(inplace=Scalar('bool'))
```

Comment:
Probably a better idea to import bool from scalar than to recreate it.  You can import it as `bool_t` so as not to shadow the python bool.